### PR TITLE
Remove namespaces, add permanent/provisional/experimental

### PR DESCRIPTION
**Pull request by cabo**
3.5 h left...

### CODE REVIEW

File: draft-ietf-sedate-datetime-extended.md [hunk 1]
```diff
@@ -245,89 +245,41 @@ first. [^interop1]
...
+  For permanent tag keys, this includes a full specification.
+  For provisional tag keys, there is an expectation that some
+  information is available even if that does not amount to a full
+  specification; in this case, the registrant is expected to improve this
+  information over time.
+
+Key names that start with an underscore are intended for experiments
+in controlled environments and cannot be registered; such keys MUST
+NOT be used for interchange and MUST be rejected by implementations
+not specifically configured to take post in such an experiment.
```
Comment (gibson042) [inline draft-ietf-sedate-datetime-extended.md:280]:
```suggestion
not specifically configured to take part in such an experiment.
```

File: draft-ietf-sedate-datetime-extended.md [hunk 2]
```diff
@@ -401,27 +353,25 @@ account.
...
 Each entry in the registry shall consist of the information described in {{registered}}.
-Initial contents of the registry are specified in {{namespaced}}.
-
-The policy is "RFC required", "Specification Required", ???[^policy]
-{{RFC8126}}.
-
-[^policy]: We need to define the policy for both namespaces and full keys.
-{: source="--- cabo"}
-
+Initial contents of the registry are specified in {{registered}}.
```
Comment (gibson042) [inline draft-ietf-sedate-datetime-extended.md:369]: Is it intentional for the referenced section to not actually define any tag keys, even the "u-ca" used in the third example?

Comment (cabo) [reply draft-ietf-sedate-datetime-extended.md:369]: No, this is a gaping hole. Added a TODO in the text.

**Review by ryzokuken — COMMENTED** [review]
Apologies for the late review, LGTM.

**Pull request merged**